.circleci/config.yml

```diff
@@ -28,6 +28,36 @@ jobs:
               exit 1
             fi
 
+  integration-test-install-scanner:
+    executor: orb-tools/ubuntu
+    steps:
+      - checkout
+      - lacework/install-lw-scanner
+      - run:
+          name: Verify install
+          command: |
+            if ! lw-scanner version; then
+              echo "The lacework inline scanner was not found"
+              exit 1
+            fi
+
+  integration-test-execute-inline-scan:
+    working_directory: ~/project
+    machine: true
+    steps:
+      - checkout
+      - lacework/install-lw-scanner
+      - run:
+          name: Pull in vuln image for testing
+          command: |
+            docker pull chriscircleci/vuln_django:7f7305bea884b63578c28d4513a916e2247ec152
+      - lacework/inline-scanner-scan:
+          registry: ""
+          repository: ""
```

Comment:
what are these two parameters for?

Reply:
@aedifex to be removed

Reply:
Good catch - @afiune @ipcrm. It's been removed.